keyboards/hillside/48/keymaps/vial/keymap.json

```
@@ -0,0 +1,88 @@
+{    "version": 1,
+    "notes": "",
+    "documentation": "\"This file is a QMK Configurator export. You can import this at <https://config.qmk.fm>. It can also be used directly with QMK's source code.\n\nTo setup your QMK environment check out the tutorial: <https://docs.qmk.fm/#/newbs>\n\nYou can convert this file to a keymap.c using this command: `qmk json2c {keymap}`\n\nYou can compile this keymap using this command: `qmk compile {keymap}`\"\n",
+    "keyboard": "hillside/48/0_1",
+    "keymap": "default",
```

```suggestion
{
    "keyboard": "hillside/48/0_1",
    "keymap": "vial",
```
See previous comment on the `hillside/46/0_1` keymap.